### ISSUE TITLE
flake8 does not generate error when given a non-existent file on the command line

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#645]
In GitLab by @diekhans on Feb 13, 2018, 24:47

*Please describe how you installed Flake8*
```
  pip install flake8
```

  into user-compiled python

*Please provide the exact, unmodified output of `flake8 --bug-report`*

```
{
  "dependencies": [
    {
      "dependency": "setuptools",
      "version": "38.5.0"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.6.2",
    "system": "Linux"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.3.1"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "1.6.0"
    }
  ],
  "version": "3.5.0"
}
```

*Please describe the problem or feature*

Running fake8 on a non-existent file does not generate an error

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

```
flake8 bin/fred
flake8 foo.py
```

where bin/fred and foo.py

no error is reported, flake8 exits with 0

**Comment by asottile** [comment]
In GitLab by @shukron on Feb 19, 2018, 12:53

This is *definitely* a bug that should be fixed. It just doesn't make sense to exit with `0` in such case

**Comment by asottile** [comment]
In GitLab by @shukron on Feb 21, 2018, 08:22

mentioned in merge request !227

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Apr 15, 2018, 16:33

closed via commit 52210149474a876c06922ae2c296445af5bdb108
